source/manual/make-github-repo-private.html.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Make a GitHub repo private
+parent: /manual.html
+layout: manual_layout
+section: GitHub
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-08-08
+review_in: 6 months
+---
+
+In cases of politically sensitive changes, we want to work in private rather than public. We can do this by creating a private fork of the repo, but this can be challenging if we need to deploy before the code can be made public. Another option is to make the repo private for a short period of time while we're working on it.
```

Comment:
```suggestion
In cases of security issues or politically sensitive changes, we want to work in private rather than public. We can do this by creating a private fork of the repo, but this can be challenging if we need to deploy before the code can be made public. Another option is to make the repo private for a short period of time while we're working on it.
```

Reply:
@tijmenb I didn't add "security issues" because these docs suggest there is a different process for that - is that no longer correct? https://docs.publishing.service.gov.uk/manual/deploy-fixes-for-a-security-vulnerability.html

Reply:
Ah, I had no idea! We should replace that section with a link to this because this is easier, but we can do that later.